tasks.py

```diff
@@ -32,6 +33,13 @@ def flake(ctx):
     ctx.run('flake8 .', echo=True)
 
 @task
+def benchmark(ctx):
+    """Run fast benchmark on codebase."""
+    # Explicitly shell out to get more consistent results by creating a new process
+    # every time, for example running out of process ensures a pristine class_registry.
+    ctx.run('python performance/benchmark.py --iterations=100 --repeat=3', echo=True)
```

Comment:
@sloria You had concerns about shelling out here.  I added a comment as to why running benchmarks out of band helps.